src/braintree/google/braintreeOnPaymentDataChangeGoogle.ts

```diff
@@ -18,13 +21,18 @@ export async function braintreeOnPaymentDataChangeGoogle(intermediatePaymentData
     const {callbackTrigger, shippingAddress, shippingOptionData} = intermediatePaymentData;
     const paymentDataRequestUpdate: PaymentDataRequestUpdate = {};
     const intent = callbackTrigger === braintreeConstants.GOOGLEPAY_TRIGGER_INITIALIZE ? braintreeConstants.GOOGLEPAY_INTENT_SHIPPING_ADDRESS : callbackTrigger as CallbackIntent;
+    const {general_settings} = getOrderInitialData();
+    const rsaEnabled = general_settings.checkout_process.rsa_enabled;
 
     switch (callbackTrigger) {
         case braintreeConstants.GOOGLEPAY_TRIGGER_INITIALIZE:
         case braintreeConstants.GOOGLEPAY_INTENT_SHIPPING_OPTION:
         case braintreeConstants.GOOGLEPAY_INTENT_SHIPPING_ADDRESS: {
-            if (shippingAddress) {
-                const address = formatBraintreeShippingAddressGoogle(shippingAddress, true);
+            let shippingLinesResponse;
+            const address = formatBraintreeShippingAddressGoogle(shippingAddress, true);
+            if (rsaEnabled) {
+                shippingLinesResponse = await estimateShippingLines(address, API_RETRY);
```

Comment:
re: @Shagufa92 comment probably applies here too

Reply:
Thank you for catch this, fixed as well